lib/datadog/core/crashtracking/agent_base_url.rb

```diff
@@ -7,10 +7,17 @@ module Core
     module Crashtracking
       # This module provides a method to resolve the base URL of the agent
       module AgentBaseUrl
+        # IPv6 regular expression from
+        # https://stackoverflow.com/questions/53497/regular-expression-that-matches-valid-ipv6-addresses
```

Comment:
https://ihateregex.io/expr/ipv6/ 😆

Reply:
I did actually look at that and it doesn't appear to work right.